time.js

```diff
@@ -0,0 +1,83 @@
+/*************
+
+MAIN LOGIC FOR CALCULATING NEXT TRAIN ARRIVAL AND MINUTES AWAY 
+GIVEN FTT (FIRST TRAIN TIME) AND FREQUENCY
+
+**************/
+
+// Utility function to add "0" in front of the hour or minute if it is less than 10 for display
+// E.g. instead of displaying 1:5 it displays 01:05
+function addZero(i) {
+	if (i < 10) {
+	    i = "0" + i;
+	}
+	return i;
```

Comment:
It is better to assign a variable to a function argument as it is bad practice to mutate and then return a function parameter, do something like:
```javascript
var result = i
//some logic
return result
```